src/Components/About/AboutControl.jsx

```diff
@@ -16,6 +16,15 @@ import PkgJson from '../../../package.json'
 export default function AboutControl() {
   const isAboutVisible = useStore((state) => state.isAboutVisible)
   const setIsAboutVisible = useStore((state) => state.setIsAboutVisible)
+
```

Comment:
This was merged in the previous PR right?

Reply:
Hadn't done the merge.  Fixed